### PR TITLE
Add a new parameter useFE to decide whether to use FE, and implement FE

### PR DESCRIPTION
**Pull request by yxd126**
The detail of useFE has been included in the README file.

### CODE REVIEW

File: ner/src/main/lbj/LbjTagger.lbj [hunk 1]
```diff
@@ -54,8 +61,12 @@ real% WordEmbeddingFeatures(NEWord word) <-
...
   			double[] embedding=WordEmbeddings.getEmbedding(w);
 			if(embedding!=null)
-				for(int dim=0;dim<embedding.length;dim++)
-					sense "place"+i+"dim"+dim : embedding[dim];
+				for(int dim=0;dim<embedding.length;dim++){
+				    sense "place"+i+"dim"+dim : embedding[dim];
+				    if(ParametersForLbjCode.currentParameters.useFE) {
+    				    sense word.domainName+"place"+i+"dim"+dim : embedding[dim];
+    				}
+				}
```
Comment (danyaljj) [inline ner/src/main/lbj/LbjTagger.lbj]: fix formatting

File: ner/README.md [hunk 2]
```diff
@@ -172,6 +172,20 @@ If set to "true", this may potentially add some robustness if you add
 sophisticated features that depend on deeper levels of NLP processing
 (currently, very few are used/required). 
 
+
+### How to use the new parameter "useFE"
+useFE is used to decide whether we want to use Frustratingly Easy Domain Adaptation or not. The default value is false. If useFE is true, then the structure of the training data folder must follow the structure like
```
Comment (danyaljj) [inline ner/README.md]: inside single quotes: `useFE`?

File: ner/src/main/lbj/LbjTagger.lbj (hunk 1 above)
Comment (yxd126) [inline ner/src/main/lbj/LbjTagger.lbj]: I have checked the code in local. All the format is correct. I don't know why it becomes this here. I will fix them manually.

**Comment by danyaljj** [comment]
@yxd126 could you make sure TeamCity is not failing.

File: ner/README.md [hunk 3]
```diff
@@ -172,6 +172,20 @@ If set to "true", this may potentially add some robustness if you add
 sophisticated features that depend on deeper levels of NLP processing
 (currently, very few are used/required). 
 
+
+### How to use the new parameter 'useFE'
+useFE is used to decide whether we want to use Frustratingly Easy Domain Adaptation or not. The default value is false. If useFE is true, then the structure of the training data folder must follow the structure like
```
Comment (danyaljj) [inline ner/README.md]: I think you have to give more details. Like you mention how things should be different for training. How about a user who doesn't care about training; does it make any different for such user?

File: ner/src/main/java/edu/illinois/cs/cogcomp/ner/ParsingProcessingData/TaggedDataReader.java [hunk 4]
```diff
@@ -77,6 +95,24 @@ public static NERDocument readFile(String path, String format, String documentNa
         return res;
     }
 
+    public static NERDocument readFile(String path, String format, String documentName, String domainName)
```
Comment (danyaljj) [inline ner/src/main/java/edu/illinois/cs/cogcomp/ner/ParsingProcessingData/TaggedDataReader.java]: Add javadocs to the input arguments of this function?

**Pull request merged**